yt_dlp/extractor/bilibili.py

```diff
@@ -754,14 +862,34 @@ class BiliBiliPlayerIE(InfoExtractor):
     _VALID_URL = r'https?://player\.bilibili\.com/player\.html\?.*?\baid=(?P<id>\d+)'
     _TEST = {
         'url': 'http://player.bilibili.com/player.html?aid=92494333&cid=157926707&page=1',
-        'only_matching': True,
+        'info_dict': {
+            'id': '92494333_part1',
+            'ext': 'm4s',
+            'uploader': '骨头゛',
+            'uploader_id': '165511',
+            'timestamp': 1583222426,
+            'title': '大学路上的绊脚石',
+            'upload_date': '20200303',
+            'thumbnail': r're:^https?://.*\.(jpg|jpeg|png)$',
+            'duration': 12.167,
+            'cid': '157926707',
+            'bv_id': 'BV1xE411H755',
+            'tags': list,
+            'description': 'md5:3c7b36418519dd8358c0aa00ab778257',
+        },
+        'params': {
+            'skip_download': True,
+        },
     }
 
     def _real_extract(self, url):
-        video_id = self._match_id(url)
-        return self.url_result(
-            'http://www.bilibili.tv/video/av%s/' % video_id,
-            ie=BiliBiliIE.ie_key(), video_id=video_id)
+        aid = self._match_id(url)
+
+        bv_id = traverse_obj(
+            self._download_json(f'https://api.bilibili.com/x/web-interface/view?aid={aid}', aid),
+            ('data', 'bvid'))
+        return self.url_result(f'http://www.bilibili.com/video/{bv_id}/',
+                               ie=BiliBiliIE.ie_key(), video_id=bv_id)
```

Comment:
Why are we updating this? As far as I know this works fine 🤔

Reply:
You are right, it works on binary release.
By the way, `http://www.bilibili.tv/video/av92494333/` has 301 Moved Permanently now.

But it do not work when I run test on src, I don't know why.
I will revert this changes